pkg/controllers/deprovisioning/suite_test.go

```diff
@@ -79,6 +81,9 @@ func TestAPIs(t *testing.T) {
 	RunSpecs(t, "Deprovisioning")
 }
 
+var benchmarkSampleSize = 20
```

Comment:
can we crank this up?

Reply:
I was considering adding a CLI flag so we can add this in. ATM I was thinking we would plug this into our CI checks, and with just these 5 tests, the benchmarking takes about 5 minutes with a sample size of 20. What are you thinking we should be having this sample size at?

Reply:
I'm not sure it makes sense to include this in our CI checks on github. GHA runners vary a ton in execution speed so I suspect the output would be all over the place. I think these benchmarks might be good to run in our e2e test cluster where we know there is some uniformity to the expected execution speed.  

I was thinking at least 100 samples.

Reply:
I'm okay, playing around with this config. I see you added to deflake, so we can take it for a spin and see what numbers we get. 👍

Reply:
> GHA runners vary a ton in execution speed

Really? They are all over the place? Do we have data around this? I'm surprised. What might be nice is to get a diff between the previous PR and this PR and do a comp, at least that's what I had in my head